pyiode/pyiode_exec.pyx

```diff
@@ -6,18 +6,109 @@
 #  IODE identity execution
 #  -----------------------
 # 
-#  TODO
-#  ---- 
-#   idt_execute(idt_list: Union(str, List[str]), sample: Union(str, List[str]), var_files: Union(str, List[str]), scl_files: Union(str, List[str])) | Execute a list of identities on a given sample
+#   idt_execute(sample: Optional[Union[str, List[str]]] = None, idt_list: Optional[Union[str, List[str]]] = None, var_files: Optional[Union[str, List[str]]] = None, scl_files: Optional[Union[str, List[str]]] = None,trace: int = 0) | Execute a list of identities on a given sample
 
+cdef extern from "iode.h": 
```

Comment:
Please move
```C
cdef extern from "iode.h": 
    ....
```
to the corresponding .pxd file

Reply:
missed my comment ?

Reply:
There is no pyiode_exec.pxd file for the moment. Must be changed later.